CK2Plus_expanded/history/provinces/1660 - Chlisselbourg.txt

```diff
@@ -0,0 +1,28 @@
+# 1660 - Chlisselbourg
+
+# County Title
+title = c_chlisselbourg
+
+# Settlements
+max_settlements = 1
+
+b_chlisselbourg = tribal	#1323

```

Comment:
What are these numbers I see popping up in these files?

Reply:
Looks like the historical founding date of the holding. Chlisselbourg seems to refer to Shlisselburg (had to pull up the wiki article in French to see the name as "Chlisselbourg"). It was founded as a fort (called Oreshek) at the head of the Neva river in 1323.

Reference: https://en.wikipedia.org/wiki/Shlisselburg